src/components/Dropdown/Dropdown.tsx

```diff
@@ -152,14 +156,9 @@ export const Dropdown: FC<DropdownProps> = React.memo(
                 { [styles.disabled]: disabled },
             ]);
             return cloneElement(child, {
-                ...{ [TRIGGER_TO_HANDLER_MAP_ON_ENTER[trigger]]: toggle(true) },
-                onClick: toggle(!mergedVisible),
                 className: referenceWrapperClasses,
-                'aria-controls': dropdownId,
-                'aria-expanded': mergedVisible,
-                'aria-haspopup': true,
```

Comment:
We seem to be losing these aria attrs. Can you please check if we need to handle them?